src/cases/store/effects/share-case.effects.ts

```diff
@@ -72,4 +74,20 @@ export class ShareCaseEffects {
       );
     })
   );
+
+  @Effect()
+  public assignUsersWithCases1$ = this.actions$.pipe(
+    ofType(shareCaseActions.ASSIGN_USERS_TO_CASE),
+    map((action: shareCaseActions.AssignUsersToCase) => action.payload),
+    switchMap(payload => {
+      this.payload = payload;
+      const ret: Observable<SharedCase[]> = this.caseShareService.assignUsersWithCases(payload);
+      return ret.pipe(
```

Comment:
not using the reference ret further down. So, can club both the statments and get rid of ret

Reply:
Defined a separate one to print console.log, forgot to remove it..resilved now